bootstrap/src/lib/util.ts

```diff
@@ -24,6 +24,14 @@ export const toObjectWithNumbers = (obj: any) => {
   return Object.fromEntries(Object.entries(obj).map(([k, v]) => [k, toNumber(v)]))
 }
 
+// pick a random string from env var after splitting with the delimiter ("a&b&c" "&" -> choice(["a","b","c"]))
+export const getRandomEnv = (name: string, delimiter = ',', prefix = '') => {
```

Comment:
Maybe we could avoid some duplication with the other method? Like passing a boolean value e.g `= (name: string, required = true, delimiter = ',', prefix = '')` and use `util.getRandomEnv('API_KEY')` in all places.

Then we can call `util.getRandomEnv('API_KEY', false)` in places that we don't want the var to be required.

Or something similar.

Reply:
I was thinking about that, but decided against it because the pattern of `getEnv` and `getRequiredEnv` already exists.

And making that change touches most External Adapters. I think we should get this in and open a ticket for that. Maybe it could still sneak in for this release.